src/electron/routing_service.ts

```diff
@@ -213,7 +216,30 @@ export class RoutingDaemon {
     return this.disconnected;
   }
 
-  public set onNetworkChange(newListener: ((status: TunnelStatus) => void)|undefined) {
+  public set onNetworkChange(newListener: ((status: TunnelStatus) => void) | undefined) {
     this.networkChangeListener = newListener;
   }
+
+  private isDaemonInstalled(): Promise<boolean> {
+    return new Promise<boolean>(resolve => {
+      const connection: Socket = createConnection(SERVICE_NAME, () => connection.destroy());
+      connection.once("error", () => connection.destroy());
+      connection.once("close", hasError => resolve(!hasError));
+    });
+  }
+
+  private installDaemon(): Promise<void> {
+    console.info(`(re-)installing routing daemon`);
```

Comment:
I'd simplify the text:
```suggestion
    console.info(`Installing routing daemon`);
```